### DISCUSSION TITLE
Problem with redirect url

### DISCUSSION THREAD

**Discussion by AmadorSV** [simpleidserver/SimpleIdServer#588] [Q&A] [answered]
Hello friends, I beeng facing some issues with the linux installation, I changed all the urls to http:// and to the ip of the linux server that I'm running SimpleId. My problem is that when I click on the Authenticate button, I get the next error: 
{
  "error": "invalid_request",
  "error_description": "redirect_uri http://myserverip:5001/signin-oidc is not correct",
  "state": "CfDJ8Fc8k3uiLCZDhN2m6XrG-6tIBdtBkfgIPrrA19jJkdH8RroLxAsZRF8WaADzgh1amBmLXIXQuZOhkBA7KpfmEm5z5D2vgqpdvrn7EmkBKyAPUQHhMtgB7NYiZuCwGmrBoizBKX4BJHZ0QlfmtM1FwZRyQgQjAtNVradrQKr3sFmnjqPbzrEbggAN3nmqwnR0n4WWKqim2K5j7ka5HObjrFUX3AGFbjrtNAPLZ49oX_eLlZ8UrZZba9EZfDxynCV7WHOLAn7jdLx6D6XLsBhlCyKT0XbhoRcw38_rij0IFS11wqBsLOF3bqRHLVBoKiwJrkfw6JZWzVYXC-aTR1_9A3mxTuFw_rKs7GATVoONxU8dde3Uy_Vk-OZEVSJIjGmNamwlpTmHIXyI3rspVoOjgQfSWtcRVXU6ub6rZZzL36GT"
}
Any Idea?

**Comment by simpleidserver** [comment, marked as answer]
Hello,

The redirect_url is automatically configured during the startup of the Identity Server. If needed, it can be updated in the database by modifying the redirect_url of the 'website' client with the correct value.

Best regards,

SID

**Comment by simpleidserver** [comment]
The documentation will be updated to explain how to configure the `redirect_url`, the ticket is #590 : https://github.com/simpleidserver/SimpleIdServer/issues/590
